cloud/aws/modules/ecs_fargate_service/main.tf

```diff
@@ -211,6 +211,10 @@ resource "aws_lb_listener" "lb_http_listeners" {
   }
 
   tags = var.tags
+
+  depends_on = [
+    aws_security_group_rule.ingress_through_http[0]
+  ]
```

Comment:
Does depends on require at least one to be present or will it work if there are no listeners?

Reply:
I tried with enable_http set to false and it still works because it won't try to create this listener if that is set to false